cmd/state-svc/internal/rtwatcher/watcher.go

```diff
@@ -100,6 +101,30 @@ func (w *Watcher) RecordUsage(e entry) {
 	w.an.EventWithSource(anaConst.CatRuntimeUsage, anaConst.ActRuntimeHeartbeat, e.Source, e.Dims)
 }
 
+func (w *Watcher) GetProcessesInUse(execDir string) []entry {
+	inUse := make([]entry, 0)
+
+	execDir = strings.ToLower(execDir) // match case-insensitively
+	for _, proc := range w.watching {
+		if !strings.Contains(strings.ToLower(proc.Exec), execDir) {
+			continue
+		}
+		isRunning, err := proc.IsRunning()
+		if err != nil && !errs.Matches(err, &processError{}) {
```

Comment:
```suggestion
		if err != nil {
```

I'm not sure what processError was meant to achieve, but it's just wrapping an OS error which we don't really know anything about, so we shouldn't treat this any different.

Reply:
I'm copying what we're doing here: https://github.com/ActiveState/cli/blob/5a71fb6fc2c1b4afff59987813ee11a9a4ebc5da/cmd/state-svc/internal/rtwatcher/watcher.go#L83.

We don't want to spam rollbar with OS-level errors (e.g. permission, process/file not found, etc.) reported by the `process` package.